template/python/cargo_version.py

```diff
@@ -8,20 +8,21 @@
 # Change versions of cargo projects.
 #
 # optional arguments:
-#   -h, --help            show this help message and exit
+#   -h, --help      show this help message and exit
 #   -p PROJECT, --project PROJECT
-#                         Project folder
-#   -r, --release         Version
+#             Project folder
+#   -r, --release     Version
 #   -n {major,minor,patch}, --next {major,minor,patch}
-#                         Version
-#   -s SET, --set SET     Version
-#   -o, --show            Version
+#             Version
+#   -s SET, --set SET   Version
+#   -o, --show      Version
 #   -m PRERELEASE, --prerelease PRERELEASE
-#                         Set pre-prelease string.
+#             Set pre-prelease string.
 #
```

Comment:
This looks badly formatted in file view?

Reply:
Thanks, I removed those